test_mail_template_attachment_i18n/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+{
+    'name': ' Tests for mail_template_attachment_i18n',
+    'summary': 'Test suite for mail_template_attachment_i18n.',
+    'author': 'Onestein,Odoo Community Association (OCA)',
+    'website': 'https://github.com/oca/',
```

Comment:
Website should be replaced by this [one](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10)